test/core/test_dsio.py

```diff
@@ -10,16 +10,37 @@
 import pandas as pd
 import xarray as xr
 
+from test.s3test import MOTOSERVER_ENDPOINT_URL, S3Test
 from test.sampledata import new_test_dataset
 from xcube.core.dsio import CsvDatasetIO, DatasetIO, MemDatasetIO, Netcdf4DatasetIO, ZarrDatasetIO, find_dataset_io, \
-    query_dataset_io, get_path_or_obs_store, write_cube, split_obs_url, parse_obs_url_and_kwargs
+    query_dataset_io, get_path_or_obs_store, write_cube, split_obs_url, parse_obs_url_and_kwargs, open_cube
 from xcube.core.dsio import open_dataset, write_dataset
 from xcube.core.new import new_cube
 
 TEST_NC_FILE = "test.nc"
 TEST_NC_FILE_2 = "test-2.nc"
 
 
+@unittest.skip('Execute manually to test with cube on S3, requires S3 credentials')
+class OpenDatasetTest(unittest.TestCase):
+    CUBE = 'eurodatacube-test/xcube-gen-f8207f0e-636d-4267-8ac5-abbdc5d121fc.zarr'
```

Comment:
I think, we should rename cubes which are used for unit-testing. It might happen, that this cube might be deleted at some point, as it also might be one of our cubes created only for gui test. I can do the renaming.

Reply:
The test is skipped. In this form, it is a deveper tool. Developers must outcomment the first line and adjust the cube path for their experiments.